website/releases/release-0.11.0.md

```diff
@@ -201,6 +201,17 @@ detailed settings.
 In 0.11.0, `org.apache.hudi.utilities.schema.HiveSchemaProvider` is added for getting schema from user-defined hive
 tables. This is useful when tailing Hive tables in `HoodieDeltaStreamer` instead of having to provide avro schema files.
 
+## Known Regression
+
+In 0.11.0 release, with the newly added support for Spark SQL features, the following performance regressions were
+inadvertently introduced:
+* Partition pruning for some of the COW tables is not applied properly
+* Spark SQL query caching (which caches parsed and resolved queries) was not working correctly resulting in additional
+* overhead to re-analyze the query every time when it's executed (listing the table contents, etc.)
+
```

Comment:
In release 0.11.0 we also have a critical regression for data loss: https://github.com/apache/hudi/pull/6179 has fixed it. Need to address it here.

Reply:
@danny0405 could you put up a PR to add the information to the release notes?

Reply:
Yeah, would fire a PR soon ~